distributions/LEIoT/options

```diff
@@ -0,0 +1,272 @@
+### DISTRO INFORMATION ###
+
+# Distro name (full name, without special characters)
+  DISTRONAME="LEIoT"
+
+# Short distro description
+  DESCRIPTION="LibreELEC for IoT (LEIoT) is a simple appliance OS for containers."
+
+# Distribution Source location                                                            
+  DISTRO_MIRROR="https://src.leiot.org/mirror"
+  DISTRO_SRC="https://src.leiot.org/$DISTRO_VERSION"
+
+# Distribution Home URL
+  DISTRO_HOME_URL="https://leiot.org"
+
+# Welcome Message for SSH Server (up to 5 Lines)
+  GREETING0="##############################################"
+  GREETING1="#                   LEIoT                    #"
+  GREETING2="#             https://leiot.org              #"
+  GREETING3="##############################################"
+  GREETING4=""
+
+# Root password to integrate in the target system
+  ROOT_PASSWORD="libreelec"
```

Comment:
Suggest changing the default password

Reply:
I've kept it the same intentionally. LEIoT is "LibreELEC" for IoT, and there's considerable muscle memory with the current LE password. Also because `leiot` is only 5-characters and less secure than `libreelec` at 9-characters. I do think we need to force change on first login though, which I think can be done with busybox. I'll look into that.